cg/cli/set.py

```diff
@@ -240,3 +240,7 @@ def microbial_sample(context, apptag, sample_id, user_signature):
             sample_obj.comment += '\n' + f"{timestamp}: {comment}"
         click.echo(click.style(f"Comment added to sample {sample_obj.internal_id}", fg='green'))
         context.obj['status'].commit()
+
+        print(click.style('update LIMS/application', fg='blue'))
+        if context.obj.get('lims'):
```

Comment:
```suggestion
        if context.obj['lims']:
```

Reply:
It for testing without having to have lims